packages/terra-table/src/terra-dev-site/doc/table/Table.1.doc.mdx

```diff
@@ -0,0 +1,82 @@
+import { Badge } from 'terra-table/package.json?dev-site-package';
+import Default from 'terra-table/lib/terra-dev-site/doc/example/DefaultTable?dev-site-example';
+import CompactPaddingTable from 'terra-table/lib/terra-dev-site/doc/example/CompactPaddingTable?dev-site-example';
+import StandardPaddingTable from 'terra-table/lib/terra-dev-site/doc/example/StandardPaddingTable?dev-site-example';
+import StripedTable from 'terra-table/lib/terra-dev-site/doc/example/StripedTable?dev-site-example';
+import CustomHeaderFooterNodeTable from 'terra-table/lib/terra-dev-site/doc/example/CustomHeaderFooterNodeTable?dev-site-example';
+import ScrollingTable from 'terra-table/lib/terra-dev-site/doc/example/ScrollingTable?dev-site-example';
+import PropsTable from 'terra-table/src/Table?dev-site-props-table';
+
+<Badge />
+
+# Terra Table
+
+The Terra Table is a structural component used to create data tables. This table also provides means of row selection and hooks for sortable columns.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-table`
+
+<!-- AUTO-GENERATED-CONTENT:START Peer Dependencies -->
+<!-- AUTO-GENERATED-CONTENT:END -->
+
+## Usage
+
+```jsx
```

Comment:
If you put this example in a jsx component and pulled it in with the codeblock loader you could get linting for free.
https://engineering.cerner.com/terra-dev-site/getting-started/terra-dev-site/webpack-loaders/codeblock-loader#codeblock-loader

I wonder if we should add a prop to the example loader to have the code open by default.